### DISCUSSION TITLE
find row by columns [Jul 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2616] [Discord Threads]
> 🙋‍♂️ **Jaxx**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### See https://docs.nocodb.com/developer-resources/rest-apis#comparison-operators  
#  
> 🙋‍♂️ **Jaxx**  
#### Thank you  
#  
> 🙋‍♂️ **Jaxx**  
#### ^ By the way this is for the API generated by NocoDB to interact with my database  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/992019068330725426)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
